hsctl

```diff
@@ -94,20 +92,20 @@ solr_schema_hs() {
     echo
     echo "*** Setting up solr schema ***"
     echo "*** From hydroshare: build_solr_schema ***"
-    docker exec hydroshare_hydroshare_1 python manage.py build_solr_schema -f schema.xml
+    docker exec hydroshare python manage.py build_solr_schema -f schema.xml
     sleep 1s
     echo "*** From solr: mv schema.xml to /opt/solr/example/solr/collection1/conf/ ***"
-    docker exec hydroshare_solr_1 cp /hydroshare/schema.xml /opt/solr/example/solr/collection1/conf/schema.xml
+    docker exec solr cp /hydroshare/schema.xml /opt/solr/example/solr/collection1/conf/schema.xml
 }
 
 start_hs() {
     echo "*** ${1^^}: all docker containers ***"
     preflight_hs
     for f in "${OTHER_DOCKER_CONTAINERS[@]}"; do
-        docker start $f;
+        docker-compose start $f;
```

Comment:
@mjstealey Didn't we have a problem using docker-compose where hsctl restart would result in the celery containers losing contact with the hydroshare container?  Have you verified that these new changes don't cause a regression?